src/index.js

```diff
@@ -117,22 +128,53 @@ function check(autoCheckElement: AutoCheckElement) {
     input.setCustomValidity('Verifying…')
   }
   autoCheckElement.dispatchEvent(new CustomEvent('loadstart'))
-  performCheck(input, body, src)
+
+  const options: RequestOptions = {body, method: 'POST'}
+
+  // If there is a controller, it means we are already in flight.
+  // Cancel that request and create a new signal.
+  let controller = abortControllers.get(autoCheckElement)
+  if (controller) {
+    // Cancel the in-flight request.
+    controller.abort()
+  }
+
+  // We need to create a new controller so we can get a new signal?
```

Comment:
The previous signal will return `true` in response to `signal.aborted`, so this comment can be removed. It's just how controllers and signals are expected to work.

Reply:
Sounds good! This was a note to my self while I was working on the PR.